action.yml

```diff
@@ -54,6 +54,10 @@ runs:
       with:
         terraform_version: ${{ inputs.terraform_version }}
 
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
     - uses: cloudposse/github-action-setup-atmos@1.0.2
```

Comment:
Can we do this?

```suggestion
    - uses: cloudposse/github-action-setup-atmos@v1
```

Reply:
Looks like we have the tag https://github.com/cloudposse/github-action-setup-atmos/tags